### PR TITLE
Added react-responsive library to determine if it is mobile screen or …

### PR DESCRIPTION
**Pull request by AdnanLatif**
…desktop

- Used react-responsive library to get device width and check if it is mobile than show text "Join now" if not mobile device than show text "Become a member"

### CODE REVIEW

File: src/screens/home/components/members-list.tsx [hunk 1]
```diff
@@ -45,7 +49,10 @@ export function MembersList(props: MembersListProps) {
           { isExpanded && <>Show less &uarr;</> }
         </Button>
         <Link href='/join'>
-          <a><Text as='span' color='gray.400' ml='20px'>Become a member</Text></a>
+          <a><Text as='span' color='gray.400' ml='20px'>
+            { isDesktopOrLaptop && <>Become a member</> }
+            { !isDesktopOrLaptop && <>Join now</> }
```
Comment (tauqeernasir) [inline src/screens/home/components/members-list.tsx]:
```suggestion
            { isDesktopOrLaptop ? 'Become a member' :  'Join now' }
```

**Review by tauqeernasir — CHANGES_REQUESTED** [review]

**Review by tauqeernasir — APPROVED** [review]
Thank you, LGTM.

**Comment by AdnanLatif** [comment]
Thank you for approving my commit.

File: .gitignore [hunk 2]
```diff
@@ -31,4 +31,7 @@ yarn-error.log*
 .env.production.local
 
 # vercel
-.vercel
+.
+
+# package-json.lock
+package-json.lock
```
Comment (nilbuild) [inline .gitignore]: Let's not ignore it

File: package.json [hunk 3]
```diff
@@ -22,6 +22,7 @@
     "react": "17.0.1",
     "react-dom": "17.0.1",
     "react-progressive-image": "^0.6.0",
+    "react-responsive": "^8.1.0",
```
Comment (nilbuild) [inline package.json]: No need for this, chakra-ui has the responsive utilities by default

File: src/screens/home/components/members-list.tsx [hunk 4]
```diff
@@ -45,7 +49,9 @@ export function MembersList(props: MembersListProps) {
           { isExpanded && <>Show less &uarr;</> }
         </Button>
         <Link href='/join'>
-          <a><Text as='span' color='gray.400' ml='20px'>Become a member</Text></a>
+          <a><Text as='span' color='gray.400' ml='20px'>
+            { isDesktopOrLaptop ? 'Become a member' :  'Join now' }
+            </Text></a>
```
Comment (nilbuild) [inline src/screens/home/components/members-list.tsx]: No need for react-responsive, we can just do
```suggestion
          <a>
            <Text d={['none', 'none', 'inline', 'inline']} as='span' color='gray.400' ml='20px'>Become a member</Text>
            <Text d={['inline', 'inline', 'none', 'none']} as='span' color='gray.400' ml='20px'>Join now</Text>
          </a>
```

**Review by nilbuild — CHANGES_REQUESTED** [review]

**Comment by AdnanLatif** [comment]
Changes in member-list file as per Kamran's suggestion

File: src/screens/home/components/members-list.tsx (hunk 4 above)
Comment (AdnanLatif) [reply src/screens/home/components/members-list.tsx]: Changes done as per your instructions.

File: package.json (hunk 3 above)
Comment (AdnanLatif) [reply package.json]: Removed react-responsive.

File: .gitignore (hunk 2 above)
Comment (AdnanLatif) [reply .gitignore]: Removed package-json.lock from .gitignore

**Review by nilbuild — APPROVED** [review]
Thanks

**Pull request merged**